contracts/EntryPoint.sol

```diff
@@ -139,7 +132,7 @@ contract EntryPoint is StakeManager {
     struct UserOpInfo {
         bytes32 requestId;
         uint256 prefund;
-        PaymentMode paymentMode;
+        address paymaster;
```

Comment:
`PaymentMode` is not used any more

Reply:
yes, it is not needed: paymaster!=address(0) means "paymaster pays".
(it used to be a 3-state, but now the mode is boolean)

Reply:
I mean, you should also remove the `enum` declaration.